github/Requester.py

```diff
@@ -833,6 +844,51 @@ def __structuredFromJson(self, data: str) -> Any:
                     raise
                 return {"data": data}
 
+    def getFile(
+        self,
+        url: str,
+        path: str,
+        parameters: Optional[Dict[str, Any]] = None,
+        headers: Optional[Dict[str, str]] = None,
+        cnx: Optional[Union[HTTPRequestsConnectionClass, HTTPSRequestsConnectionClass]] = None,
+    ) -> None:
+        """
+        GET a file from the server and save it to the given path.
+        """
+        _, _, stream_chunk_iterator = self.getResponseStream(url, parameters, headers, cnx)
+        with open(path, "wb") as f:
+            for chunk in stream_chunk_iterator():
+                if chunk:
+                    f.write(chunk)
+
+    def getResponseStream(
+        self,
+        url: str,
+        parameters: Optional[Dict[str, Any]] = None,
+        headers: Optional[Dict[str, str]] = None,
+        cnx: Optional[Union[HTTPRequestsConnectionClass, HTTPSRequestsConnectionClass]] = None,
+    ) -> Tuple[int, Dict[str, Any], Callable]:
```

Comment:
I think we can be more specific here:
```suggestion
    ) -> Tuple[int, Dict[str, Any], Callable[Union[int, None], Iterator]:
```

Reply:
that didn't work, but I implemented the spirit of the comment